### DISCUSSION TITLE
Autolinks reference to external resource in PR title

### DISCUSSION THREAD

**Discussion by kkevindev** [community/community#101658] [Repositories]
### Select Topic Area

Product Feedback

### Body

It would be nice if https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/configuring-autolinks-to-reference-external-resources would also work on the PR title (and not just the PR description).

We (developers at our company) create PR's with a title like `[DV-0000] Improved X.` where `DV-0000` would be the link to a Jira ticket. The autolinks feature works for the PR description and commit messages, but doesn't work in the PR title. 

It would be nice if the external autolinks feature would also work on PR titles.

<img width="255" alt="afbeelding" src="https://github.com/community/community/assets/34477826/355f31c6-6a71-4641-9465-6b48a96c3129">

**Comment by dgpokl** [comment]
Please, consider adding this. The PR title is the most obvious and probably one of the most common places people mention ticket numbers. It's absolutely painful to not be able to simply jump from a PR to the ticket!

**Comment by dgpokl** [comment]
Dupe of #9369 - please upvote that one as well

**Comment by twojnarowski** [comment]
I agree, it should work in titles of PRs as well, both on the list of PR and in the details of PR
